<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoanWindow</class>
 <widget class="QDialog" name="LoanWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>0</y>
     <width>161</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Loan Window</string>
   </property>
  </widget>
  <widget class="QLabel" name="AccNumLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>481</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: black;
}</string>
   </property>
   <property name="text">
    <string>Account Number: 1110-01023-4122</string>
   </property>
  </widget>
  <widget class="QLabel" name="UsernamLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>481</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: black;
}</string>
   </property>
   <property name="text">
    <string>Username:  Thanaphat</string>
   </property>
  </widget>
  <widget class="QLabel" name="BalanceLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>481</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: black;
}</string>
   </property>
   <property name="text">
    <string>Balance:   10,212.00 THB</string>
   </property>
  </widget>
  <widget class="QLabel" name="BalanceLabel_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>180</y>
     <width>481</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: black;
}</string>
   </property>
   <property name="text">
    <string>Loan Balance:   1,010.20 THB</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>390</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Withdraw to Balance</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
